src/main.ts

```diff
@@ -314,18 +314,25 @@ export default class CopilotPlugin extends Plugin {
     });
 
     this.addCommand({
-      id: "find-similar-notes",
-      name: "Find similar notes to active note",
+      id: "find-relevant-notes",
```

Comment:
I think the term "relevant" is more accurate as we are adding information that is more than just similarity